frameworks/CSharp/aspnetcore/benchmark_config.json

```diff
@@ -20,9 +19,26 @@
       "notes": "",
       "versus": ""
     },
+    "middleware-json": {
+      "setup_file": "setup-json",
+      "json_url": "/json",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "None",
+      "framework": "aspnetcore",
+      "language": "C#",
+      "orm": "Raw",
+      "platform": "NET",
```

Comment:
Reading more of the existing results, I totally left out "framework" (doh).

With that in mind, perhaps the following is better:
- language = C#
- platform = NET
- framework = aspnetcore for our tests. If Nancy submits tests, they'd be "nancy"
- classification = "Platform" for when we use Kestrel (our server) directly without middleware at all (we don't have a test for this right now), "Micro" for raw middleware or routing, "FullStack" for MVC controllers
- front-end-server = None in our case, as the framework includes a server in-process (Kestrel)
- flavor = CoreCLR or CLR  or CoreRT or Mono